<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerSetWidget</class>
 <widget class="QWidget" name="ServerSetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>图片质量：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_quality">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_quality">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>帧 间 隔：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_interval">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_interval">
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>广播地址：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combobox_broadcast">
       <property name="minimumSize">
        <size>
         <width>125</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>报文长度：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineedit_block_size"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>建议设置：256</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_start">
       <property name="text">
        <string>启动</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_stop">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
